lib/bolt/config.rb

```diff
@@ -176,6 +187,27 @@ def update_from_cli(options)
       end
     end
 
+    def update_from_inventory(data)
+      update_from_file(data)
```

Comment:
do we want to copy all the non-transport config onto the target? I guess it doesn't do anything and isn't accessible from puppet code

Reply:
Right now they're ignored, and it's a couple strings (so cost of cloning should be low). That said, I'd like to benchmark how this all affects searching a large inventory.